_posts/2023-06-19-geoserver-2-22-4-released.md

```diff
@@ -0,0 +1,112 @@
+---
+author: Peter Smythe
+date: 2023-06-19
+layout: post
+title: GeoServer 2.22.4 Release
+categories:
+- Announcements
+tags:
+- Release
+- Vulnerability
+release: release_222
+version: 2.22.4
+jira_version: 16891
+--- 
+
+GeoServer [2.22.4](/release/2.22.4/) release is now available
+with downloads (
+[bin](https://sourceforge.net/projects/geoserver/files/GeoServer/2.22.4/geoserver-2.22.4-bin.zip/download),
+[war](https://sourceforge.net/projects/geoserver/files/GeoServer/2.22.4/geoserver-2.22.4-war.zip/download),
+[windows](https://sourceforge.net/projects/geoserver/files/GeoServer/2.22.4/GeoServer-2.22.4-winsetup.exe/download))
+, along with 
+[docs](https://sourceforge.net/projects/geoserver/files/GeoServer/2.22.4/geoserver-2.22.4-htmldoc.zip/download) and
+[extensions](https://sourceforge.net/projects/geoserver/files/GeoServer/2.22.4/extensions/).
+
+This is a maintenance release of GeoServer providing a existing installations with minor updates and bug fixes.
+GeoServer is made in conjunction with GeoTools 28.4. 

```

Comment:
```suggestion
GeoServer 2.24.4 is made in conjunction with GeoTools 28.4. 

```

Reply:
@jodygarnett This should be GeoServer 2.22.4